tests/install-errors/task.yaml

```diff
@@ -1,4 +1,22 @@
 summary: Checks for cli errors installing snaps
+environment:
+  SIDELOAD_SNAP_NAME: basic-binaries
+  STORE_SNAP_NAME: hello-world
+  SNAP_FILE: "./$[SIDELOAD_SNAP_NAME]_1.0_all.snap"
+
+prepare: |
```

Comment:
I love how those tests look like. If you recall our conversation with @elopio at the sprint I argued we should do something exactly like this. This is brilliant.